tests/GUI/docBasicTests.spec.ts

```diff
@@ -0,0 +1,194 @@
+import test from "@playwright/test";
+import { LeftMenu } from "../../page-objects/leftMenu";
+import { SpaceContextMenu } from "../../page-objects/context-menus/spaceContextMenu";
+import { SpacePlusMenu } from "../../page-objects/context-menus/spacePlusMenu";
+import { ApiHooks } from "../../api-utils/apiHooks";
+import { DocView } from "../../page-objects/docView";
+import { faker } from "@faker-js/faker";
+import { ApiUtils } from "../../api-utils/apiUtils";
+import { DocContextMenu } from "../../page-objects/context-menus/docContextMenu";
+
+test.describe(
+  "UI tests for checking basic doc functionalities",
+  {
+    tag: ["@doc", "@one"],
+    annotation: {
+      type: "test type",
+      description: "One basic test",
+    },
+  },
+  async () => {
+    const newSpaceName = "GUI TEST new space";
+    let leftMenu: LeftMenu;
+    let newDocName = faker.commerce.productName();
+
+    test.beforeAll("Prepare environment before tests", async ({ request }) => {
+      await ApiHooks.createSpaceByName(request, newSpaceName);
+    });
+
+    test.afterAll("Prepare environment before tests", async ({ request }) => {
+      await ApiHooks.deleteSpaceByName(request, newSpaceName);
+    });
+
+    test.beforeEach(async ({ page }) => {
+      leftMenu = new LeftMenu(page)
+    })
```

Comment:
```suggestion
    });
```
Daj ctrl+s żeby to prettier przejechał + pytanie czy chcemy jeden page do befora? IMO to jest OK, ale nie wiem jaka byłaby najlepsza praktyka.